enterprise/server/cmd/ci_runner/main.go

```diff
@@ -697,6 +703,39 @@ func ensurePath() error {
 	return os.Setenv("PATH", "/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin")
 }
 
+func getBazeliskCommand() (string, error) {
+	wd, err := os.Getwd()
+	if err != nil {
+		return "", err
+	}
+	// If bazelisk is already in the workspace root, use that.
+	if _, err := os.Stat(bazeliskBinaryName); err != nil {
+		if !os.IsNotExist(err) {
+			return "", err
+		}
+	} else {
+		return filepath.Join(wd, bazeliskBinaryName), nil
+	}
+	// If bazelisk is embedded, copy it to the workspace root, and use that.
+	if f, err := bazelisk.Open(); err == nil {
+		defer f.Close()
+		dst, err := os.OpenFile(bazeliskBinaryName, os.O_CREATE|os.O_TRUNC|os.O_WRONLY, 0555)
+		if err != nil {
+			return "", err
+		}
+		defer dst.Close()
+		if _, err := io.Copy(dst, f); err != nil {
+			return "", err
+		}
+		return filepath.Join(wd, bazeliskBinaryName), nil
+	}
+	// If bazelisk is in PATH, use that.
+	if _, err := exec.LookPath(bazeliskBinaryName); err != nil {
+		return "", err
+	}
```

Comment:
Should we move this up?

The behavior of using the bundled bazelisk, unless there was an error, in which case we fall back to the system one, is unusual. If we used the system one and fell back to built in, that would make sense, or alternatively, if we always used the bundled one or returned an error, that would also make sense.

Reply:
sg to always use the bundled one -- done. I made bazelisk a hard dependency instead of excluding it in fastbuild mode, which seems OK since it's pretty small. If we find that this slows down builds during development then we can maybe use a fileresolver to have it work with runfiles as well.